packages/strapi-helper-plugin/lib/src/components/HeaderSearch/Wrapper.js

```diff
@@ -0,0 +1,46 @@
+import styled from 'styled-components';
+
+const Wrapper = styled.div`
+  position: fixed;
+  top: 0;
+  display: flex;
+  align-items: center;
+  overflow: hidden;
+  min-width: 44rem;
+  height: 6rem;
+  padding-right: 20px;
+  background-color: #ffffff;
+  border-right: 1px solid #f3f4f4;
+  z-index: 1050;
+  color: #9ea7b8;
+  line-height: 6rem;
+  letter-spacing: 0;
+
+  > div:first-child {
```

Comment:
I agree about the selectors however, this particular component has been moved from the content manager to the strapi-helper-plugin and since time is lacking on our hand and also since it has been already been used I won't update its styling in this particular PR but in another one when we finish stabilising the administration panel. 😉